README.md

```diff
@@ -115,6 +116,33 @@ the card has:
 The demo subdirectory includes details of a OpenVINO deployment and use of the VPU plugin.
 Sources can be found in [openvino-demo](demo/ubuntu-demo-openvino)
 
+## Operator
```

Comment:
The title could highlight that this is a unified operator for all device plugins. `Device Plugins Operator` perhaps?